### PR TITLE
Feat/550 copy mnemonic button

### PR DESCRIPTION
**Pull request by Sm00g15**

### CODE REVIEW

**Comment by Sm00g15** [comment]
I put the copy button above the regenerate mnemonic button b/c the hidden tooltip from the copied button was covering some of the regenerate button's letters

**Comment by umurb** [comment]
1. Currently, only the Icons are actionable. We need both Icons and Phrases to be actionable (clickable). My cursor should become a pointer when hovering over all these components.

2. Items are misaligned

3. Incorrect locales files are checked in. The items you are trying to merge are located in
`extraResources/locales/ ` folder whereas we keep the correct locales files under `app/extraResources/locales/`. Please fix

4. Please do not check in package-lock.json

File: app/components/CopyIcon/index.js [hunk 1]
```diff
@@ -31,6 +31,13 @@ class CopyIcon extends Component<Props> {
     showCopyConfirmation: false
   };
 
+  componentWillUpdate() {
```
Comment (developer0623) [inline app/components/CopyIcon/index.js]: Why did you use this lifecycle?

File: app/components/CreateAccountSlide/index.js [hunk 2]
```diff
@@ -156,13 +185,32 @@ class CreateAccountSlide extends Component<Props> {
...
+        <IconContainer>
+          <IconCopy>
+            <CopyIcon
+              spanClicked={spanClicked}
+              text={processedSeed}
+              color="#2c7df7"
+            />
+            <span
+              onClick={this.spanClick}
+              style={{ cursor: 'pointer', marginLeft: '3px' }}
```
Comment (developer0623) [inline app/components/CreateAccountSlide/index.js]: Don't use inline style, please.
Please use the styled component

**Pull request merged**